### PR TITLE
Fix normalization

### PR DESCRIPTION
**Pull request by TimScherr**

### CODE REVIEW

**Comment by TimScherr** [comment]
Masks:
01_mask 02_mask 03_mask

Distance to background:
01_cell 02_cell 03_cell

Distance to neighbors (old):
01_neighbor 02_neighbor 03_neighbor

Distance to neighbors (new):
01_neighbor_new 02_neighbor_new 03_neighbor_new

**Comment by TimScherr** [comment]
- Needs evaluation how results in the publication will change
- Needs evaluation on an internal, large, and diverse dataset

**Comment by TimScherr** [comment]
New version on microbeSEG data: roughly the same results as the old one (5 models trained). No large differences were expected since normalization worked well for most of this data.

New version trained on a large dataset (omnipose data, microbeSEG data, Cell Tracking Challenge data): less over-segmentation of meandering objects, raw predictions look better/sharper (as expected) --> would maybe allow fine-tuned post-processing.

Can be merged after some more tests.

**Comment by TimScherr** [comment]
Cell distances look nearly the same:

grafik

**Comment by TimScherr** [comment]
Test image from omnipose test set and predictions (old version, new version):

grafik

**Comment by TimScherr** [comment]
Image/ground truth/old version prediction/new version prediction for former shown raw predictions:

grafik
